opentelemetry-quickstart/src/main/resources/application.properties

```diff
@@ -1,5 +1,4 @@
 quarkus.application.name=myservice
-quarkus.otel.exporter.otlp.traces.endpoint=http://localhost:4317
+quarkus.otel.metrics.enabled=true
```

Comment:
Hi @jcarranzan, after another pass at the issue I found that you need to add an additional property to activate metrics for native mode. 
Please add this config and comment:
```bash
# jfr must be enabled if you need OpenTelemetry metrics on native mode.
quarkus.native.monitoring=jfr
```

This will unlock the build.
I'll deal with the documentation fix this requires.

Reply:
Yeah I was just reading about this : https://es.quarkus.io/guides/jfr#native-image
I'll push it those changes, thanks.